org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/logger/ScoutBasicLogbackLayout.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2010-2022 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+package org.eclipse.scout.rt.platform.logger;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.eclipse.scout.rt.platform.util.StringUtility;
+
+import ch.qos.logback.classic.PatternLayout;
+import ch.qos.logback.classic.pattern.ThrowableProxyConverter;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.CoreConstants;
+import ch.qos.logback.core.LayoutBase;
+import ch.qos.logback.core.util.CachingDateFormatter;
+
+/**
+ * Own layouter for logback with fixed pattern (see {@link #doLayout(ILoggingEvent)}) and additional customization of
+ * MDC handling. Allows to define renamings of MDC keys, exclusions, inclusions (the override default exclusions) and
+ * order of MDC entries.
+ * <p>
+ * Main reason to use this layout instead of e.g. {@link PatternLayout} is that all MDC context are logged by default
+ * (with default exclusion), thus not requiring to modify logback configs if a new MDC context is added that shouldn't
+ * be missed.
+ * <p>
+ * Default exclusions are
+ * <ul>
+ * <li>scout.user.name
+ * <li>http.session.id
+ * <li>http.request.method
+ * <li>http.request.uri
+ * </ul>
+ * Own exclusions can be defined with the XML tag <code>mdcExclusion</code>. Inclusions can be defined with the XML tag
+ * <code>mdcInclusion</code>
```

Comment:
```suggestion
 *
 * To output an excluded MDC value, it can be defined as an inclusion. Inclusions can be defined with the XML tag
 * <code>mdcInclusion</code.
```